bigtable/tests/system.py

```diff
@@ -244,6 +244,21 @@ def test_create_table(self):
         sorted_tables = sorted(tables, key=name_attr)
         self.assertEqual(sorted_tables, expected_tables)
 
+    def test_create_table_with_split_keys(self):
+        temp_table_id = 'foo-bar-baz-split-table'
+        initial_split_keys = [b'split_key_1', b'split_key_10',
+                              b'split_key_20', b'']
+        temp_table = Config.INSTANCE.table(temp_table_id)
+        temp_table.create(initial_split_keys=initial_split_keys)
+
+        # Read Sample Row Keys for created splits
+        sample_row_keys = temp_table.sample_row_keys()
+
+        self.tables_to_delete.append(temp_table)
+
+        for sample_row_key in sample_row_keys:
+            self.assertIn(sample_row_key.row_key, initial_split_keys)
```

Comment:
Shouldn't we be asserting that `sample_row_keys` contains the *all* the same key values as `initial_row_keys`?  E.g.:

```python
        self.assertEqual(
            set([srk.row_key for srk in sample_row_keys]),
            set(initial_split_keys))
```